build/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Changing the container's glib version might cause issues.
 # In the past, we experienced problems when bumping Go versions and using the `bookworm` versions.
 # glib issue: https://github.com/golang/go/issues/58550#issuecomment-1597411275
-FROM golang:1.19.10-buster
+FROM golang:1.22.4-bullseye
```

Comment:
Do you know if we are still affected by the glib issue when updating the container's version? (The issue pointed out in the comment above)

Reply:
it should be solved now! I'll remove the comment 😄